### ISSUE TITLE
Needs accessibility rules

### ISSUE THREAD

**Issue by COLABORATI** [NamPNQ/You-Dont-Need-Javascript#4]
Some of these examples do not work with keyboard only navigation.

**Comment by kylezinter** [comment]
A lot of these examples would not work well for those with impairments that relay on web accessibility best practices to use web sites. This should be called out in your readme.

**Comment by nico3333fr** [comment]
A lot of these examples are really funny from a CSS point of view, but are really __bad for accessibility__: 

- modals without JS don't trap the keyboard focus in them
- accordions are not working with keyboard
- other will be vocalized as form elements
- etc.

Please add a notice that these examples are proofs of concept, but really lack for accessibility.

**Comment by joe-watkins** [comment]
While many of these patterns are anti-patterns, they are pretty cool :) A little note in the readme about how they could harm accessibility could be a solve.

**Comment by Julix91** [comment]
I think the note has been added, should the topic be closed?

**Comment by stevemao** [comment]
PR welcome

**Comment by Aditya-Singh2955** [comment]
can u assign this to me ?
